### PR TITLE
Fix icons in CRX

### PR DESCRIPTION
**Pull request by galeksandrp**

### CODE REVIEW

**Comment by J0WI** [comment]
@Hainish is this fixing the problem we had in the last roll out?

**Comment by J0WI** [comment]
@galeksandrp could you add a description what this PR is fixing?

**Comment by galeksandrp** [comment]
1. `git clone https://github.com/EFForg/https-everywhere.git && cd https-everywhere`
2. `./makecrx.sh`
3. image

**Comment by J0WI** [comment]
I was not able to reproduce this on the new `webextensions` branch, but let's ping @Hainish again to ask him.

**Comment by galeksandrp** [comment]
It is reproducible only on Chrome, since git clone does not create symlinks on Windows.

File: makecrx.sh [hunk 1]
```diff
@@ -57,6 +57,10 @@ sed -e "s/VERSION/$VERSION/g" chromium/updates-master.xml > chromium/updates.xml
 mkdir -p pkg/crx/rules
 cd pkg/crx
 rsync -aL ../../chromium/ ./
```
Comment (Hainish) [inline makecrx.sh]: This `rsync` command should suffice, since the `-L` flag copies symlinks.

**Pull request closed without merging**